charts/scalardl-audit/values.yaml

```diff
@@ -0,0 +1,239 @@
+    type: RollingUpdate
+
+  # -- PodSecurityContext holds pod-level security attributes and common container settings
+  podSecurityContext:
+    {}
+    # fsGroup: 2000
+
+  # -- Setting security context at the pod applies those settings to all containers in the pod
+  securityContext:
+    {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  service:
+    # -- service types in kubernetes
+    type: ClusterIP
+    # -- Service annotations, e.g: prometheus, etc.
+    annotations: {}
+    ports:
+      envoy:
+        # -- envoy public port
+        port: 40051
+        # -- envoy k8s internal name
+        targetPort: 50051
+        # -- envoy protocol
+        protocol: TCP
+      envoy-priv:
+        # -- nvoy public port
+        port: 40052
+        # -- envoy k8s internal name
+        targetPort: 50052
+        # -- envoy protocol
+        protocol: TCP
+
+  serviceMonitor:
+    # -- enable metrics collect with prometheus
+    enabled: false
+    # -- custom interval to retrieve the metrics
+    interval: "15s"
+    # -- which namespace prometheus is located. by default monitoring
+    namespace: monitoring
+
+  prometheusRule:
+    # -- enable rules for prometheus
+    enabled: false
+    # -- which namespace prometheus is located. by default monitoring
+    namespace: monitoring
+
+  grafanaDashboard:
+    # -- enable grafana dashboard
+    enabled: false
+    # -- which namespace grafana dashboard is located. by default monitoring
+    namespace: monitoring
+
+  # -- resources allowed to the pod
+  resources:
+    {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  # -- nodeSelector is form of node selection constraint
+  nodeSelector: {}
+
+  # -- Tolerations are applied to pods, and allow (but do not require) the pods to schedule onto nodes with matching taints.
+  tolerations: []
+
+  # -- the affinity/anti-affinity feature, greatly expands the types of constraints you can express
+  affinity: {}
+
+auditor:
+  # -- number of replicas to deploy
+  replicaCount: 3
+
+  scalarAuditorConfiguration:
+    # -- The contact points of the database such as hostnames or URLs
+    dbContactPoints: cassandra
+    # -- The port number of the contact points
+    dbContactPort: 9042
+    # -- The username of the database
+    dbUsername: cassandra
+    # -- The password of the database
+    dbPassword: cassandra
+    # -- The storage of the database: cassandra or cosmos
+    dbStorage: cassandra
+    # -- The port number of Auditor Server
+    auditorServerPort: 50051
+    # -- The port number of Auditor Privileged Server
+    auditorServerPrivilegedPort: 50052
+    # -- The port number of Auditor Admin Server
+    auditorServerAdminPort: 50053
+    # -- The log level of Scalar auditor
+    auditorLogLevel: INFO
```

Comment:
Why not define a default value for `auditorLedgerHost`?
https://github.com/scalar-labs/helm-charts/pull/21/files#diff-869c5b6a2263743622afe38284e8cb9c052f52e1c223287f3d32727b8ee7dee6R94